README.md

```diff
@@ -1 +1,21 @@
-# Beyond-07-team-5
\ No newline at end of file
+
+![ResuMe](ResuMe/static/images/thumbnail.png)
```

Comment:
```suggestion
![ResuMe](./static/images/thumbnail.png)
```

Reply:
Still not working

Reply:
Check the file in the "view file" button. It looks ok
https://github.com/redhat-beyond/ResuMe/blob/3197e749e067670d1886eb19d9613fc89d4ffb7e/README.md

You can also check this in your IDE. You have a preview option for markdown files

Reply:
i can see that it working 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/70466270/157204334-a00cca92-fc2b-44a4-82f9-d10e951ba01f.png">

Reply:
![image](https://user-images.githubusercontent.com/36789773/157204346-91c1b281-7cab-4c70-94e5-a73d744e8fd1.png)

I the file changes I still can't see the picture.. 
Maybe it because the branch didn't merge?

Reply:
I see it also good in your fork [MatanP12/readme](https://github.com/MatanP12/ResuMe/tree/readme)

Reply:
Ok Thank you all!
I will squash the commits and push

Reply:
LGTM, I can also see the picture in your fork.

Reply:
wait with the squash until ll the PR is done